init

```diff
@@ -1,21 +1,40 @@
 #!/usr/bin/bash
 
+echo "-------Beginning bootstrap script-------"
+
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 HOMEDIR=$(eval echo "~`whoami`")
 
+bash ${SCRIPTDIR}/v2/util/setup-credentials.sh
+
+if [ -f /etc/environment ]; then
+    source /etc/environment
+fi
+
+if [ "$NODE_ROLE" = "control" ]; then
+  echo "-------Control node found, setting up etcd peers-------"
+  sudo bash ${SCRIPTDIR}/v2/util/etcd-peers-dropin.sh
+  sudo systemctl enable "${SCRIPTDIR}/v2/util-units/etcd-peers.service"
```

Comment:
I think that it might be better to move or `cp` this into a dropin dir in `/etc/systemd`. The repo in `/home/core` *can* be deleted at any point and `systemctl` symlinks the files, I think?